caput/memh5.py

```diff
@@ -2601,6 +2688,38 @@ def _prepare_compression_args(dset):
     return distributed_dset_names if skip_distributed else None
 
 
```

Comment:
I considered using `@overload` here, but it really doesn't look pretty. I'm open to thoughts about using something better than `Any` though